packages/terra-card/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.log
```

Comment:
In the future please don't do this unless there is a compelling reason to do so. It blows away all review comments that people have made.